templates/complete.html

```diff
@@ -13,28 +15,25 @@
         <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
     </head>
     
-    <body>
+    
         <h1>Your file has been successfully uploaded. Here is a map visualization:</h1>
  
 <div id='map' style='width: 100%; height: 500px;'></div>
 <script type="text/javascript">
+  //Mapbox access token
 	mapboxgl.accessToken = 'pk.eyJ1IjoiZnJlaWphIiwiYSI6ImNqM3FjajEycDAwbmYycW1rZDZ3Y3ZmaDgifQ.gG6EIJvRcdlHGtyaF7vyrw';

```

Comment:
Access tokens should be kept secret. For example, set it in a settings file that is not tracked by git.